CHANGELOG.md

```diff
@@ -9,6 +9,48 @@ and this project adheres to
 
 ## Unreleased
 
+### Added
+
+- Added support for `string[]` environment variables. Example:
+
+`.env` file:
+
+```
+// .env
+SEVERITIES='HIGH,CRITICAL'
```

Comment:
I know we've done it for a while, but I think we should consider supporting a valid JSON array instead:
.env:
```
SEVERITIES=["HIGH", "CRITICAL"]
```
It would just involve calling JSON.parse on the loaded string and then validating they are all strings. Wdyt?

Reply:
makes sense - we could expand this type to just be `json` and load whatever we want